### ISSUE TITLE
Unsupported PlatformType

### ISSUE THREAD

**Issue by joalisonx** [isFakeAccount/psnawp#54]
Apparently, when I try to fetch a game that has a set of trophies shared between VITA and PS3, the wrapper gives me an error: `ValueError: 'PS3,PSVITA' is not a valid PlatformType`
This game is Batman: Arkham Origins

**Comment by isFakeAccount** [comment]
@joalisonx Can you provide details on how to reproduce this error?

**Comment by joalisonx** [comment]
Sure. It also gives me the same error with PS Vita platform.
```py
client = PSNAWP(get_env()["npsso"])
game = client.game_title(title_id="NPWR04151_00", np_communication_id="NPWR04151_00").trophy_groups_summary("PS3")
```

```py
Traceback (most recent call last):
  File "/platinador/src/ext/find.py", line 26, in find
    game = get_game(game_id, platform)
  File "/platinador/src/utils/utils.py", line 35, in get_game
    raise err
  File "/platinador/src/utils/utils.py", line 26, in get_game
    game = client.game_title(title_id=game_id, np_communication_id=game_id).trophy_groups_summary(platform)
  File "/lib/python3.8/site-packages/psnawp_api/models/game_title.py", line 119, in trophy_groups_summary
    return TrophyGroupsSummaryBuilder(
  File "/lib/python3.8/site-packages/psnawp_api/models/trophies/trophy_group.py", line 162, in game_title_trophy_groups_summary
    return _trophy_groups_dict_to_obj(response)
  File "/lib/python3.8/site-packages/psnawp_api/models/trophies/trophy_group.py", line 99, in _trophy_groups_dict_to_obj
    trophy_title_platform=PlatformType(trophy_groups_dict.get("trophyTitlePlatform", "UNKNOWN")),
  File "/lib/python3.8/enum.py", line 304, in __call__
    return cls.__new__(cls, value)
  File "/lib/python3.8/enum.py", line 595, in __new__
    raise exc
  File "/lib/python3.8/enum.py", line 579, in __new__
    result = cls._missing_(value)
  File "/lib/python3.8/enum.py", line 608, in _missing_
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 'PS3,PSVITA' is not a valid PlatformType
```

**Comment by isFakeAccount** [comment]
This bug is fixed in version 1.3.2 https://github.com/isFakeAccount/psnawp/releases/tag/v1.3.2
